avocado/plugins/run.py

```diff
@@ -128,10 +128,13 @@ def configure(self, parser):
 
         if multiplexer.MULTIPLEX_CAPABLE:
             mux = parser.add_argument_group('multiplexer use on test execution')
-            mux.add_argument('-m', '--multiplex-files', nargs='*',
+            mux.add_argument('-m', '--multiplex', nargs='*', dest='multiplex_files',
                              default=None, metavar='FILE',
                              help='Location of one or more Avocado multiplex (.yaml) '
                              'FILE(s) (order dependent)')
+            mux.add_argument('--multiplex-files', nargs='*',
+                             default=None, metavar='FILE',
+                             help='DEPRECATED: please use --multiplex instead')
```

Comment:
How about (ab)using `type/action` to log into warning that this option is deprecated? Alternatively if you move away from using `dest` in `multiplex`, you can simply log the warning when processing `multiplex_files` (which would be cleaner)